src/entities/priceImpact/addLiquidityNested.ts

```diff
@@ -134,6 +134,7 @@ async function getAddBoostedUnbalancedResult(
         rpcUrl,
         amountsIn,
         kind: AddLiquidityKind.Unbalanced,
+        wrapUnderlying: [true, true],
```

Comment:
This is bad, but `wrapUnderlying` has to be there for price impact test to pass. Not sure how to handle cus adding `wrapUnderlying` to `AddLiquidityNestedInput` type would blow up nested which is not supported by the new router